pkg/server/router/manifest.go

```diff
@@ -223,7 +223,16 @@ func (mr ManifestRouter) SubmitApplication(ctx context.Context, w http.ResponseW
 		return framework.NewRequestError(errors.Wrap(err, errMsg), http.StatusInternalServerError)
 	}
 
-	resp := SubmitApplicationResponse{Response: submitApplicationResponse.Response, Credentials: submitApplicationResponse.Credential}
+	var credentials []interface{}
+	for _, container := range submitApplicationResponse.Credential {
+		if container.HasDataIntegrityCredential() {
+			credentials = append(credentials, *container.Credential)
+		}
```

Comment:
should there be an else if? There could be a case where someone fills out a JWT and a DataIntegrityCred and it would print 2 credentials

Reply:
good call, updated